tests/utils.py

```diff
@@ -64,6 +65,10 @@ def __init__(self,
                  env_dict: Optional[Dict[str, str]] = None,
                  auto_port: bool = True,
                  max_wait_seconds: Optional[float] = None) -> None:
+        if not model.startswith("/"):
+            # download the model if it's not a local path
+            # to exclude the model download time from the server start time
+            model = snapshot_download(model)
```

Comment:
snapshot_download can download many more files than we need for inference, such as duplicate .pt files when we prefer to use safetensors. We should try to use a common function with how vLLM usually pulls down files for models

Reply:
can you come up with a fix for it?

I don't know if safetensors would be enough. but we can have atry.

Reply:
Basically I am proposing using `DefaultModelLoader._prepare_weights`
https://github.com/vllm-project/vllm/blob/ea9fa160e3b47e0b8aa273f3eb2be410bd1ccab5/vllm/model_executor/model_loader/loader.py#L232
I can make a PR for this later today

Reply:
please go ahead, I do see more files are downloaded in https://buildkite.com/vllm/fastcheck/builds/3094#01918525-98da-4596-8d31-f4e2c1172455